roles/edpm_neutron_metadata/templates/neutron-ovn-metadata-agent.conf.j2

```diff
@@ -1,8 +1,18 @@
 [DEFAULT]
 debug = {{ edpm_neutron_metadata_agent_DEFAULT_debug }}
+
+{%- if edpm_neutron_metadata_agent_DEFAULT_nova_metadata_host is defined %}
```

Comment:
{%- --> {% seems to be causing it, can drop from all the places in beginning of the block

Reply:
Done